internal/config/profile.go

```diff
@@ -585,3 +611,18 @@ func configHome() (string, error) {
 
 	return fmt.Sprintf("%s/.config", home), nil
 }
+
+// AtlasCLIConfigHome retrieves configHome path based used by atlasCLI.
+func AtlasCLIConfigHome() (string, error) { return atlasCLIConfigHome() }
+func atlasCLIConfigHome() (string, error) {
```

Comment:
do you need a public and private method that both do the same?

Reply:
no, I will update this

Reply:
Updated